extension/src/test/e2e/extension.test.ts

```diff
@@ -17,6 +17,9 @@ suite('DVC Extension For Visual Studio Code', () => {
   before('should finish loading the extension', async function () {
     this.timeout(240000)
     await waitForViewContainerToLoad()
+    const workbench = await browser.getWorkbench()
+    await workbench.executeCommand('DVC: Garbage Collect Experiments')
+    await browser.keys('Enter')
```

Comment:
[F] This removes experiments that are found against previous commits. It is very likely that they will be missing from the remote so `plots diff` will fail to load the data.